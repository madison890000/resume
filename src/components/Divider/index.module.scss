.divider {
  margin: var(--block-margin) 0 0 0;
  width: 100%;
}

.section {
  margin: 10px 0;
  border: var(--base-gap-unit) solid gray;
}
