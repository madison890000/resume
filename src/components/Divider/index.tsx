import React from 'react';
import styles from './index.module.scss';

interface DividerProps {
    title?: string;
}

const Divider = ({
                     title,
                 }: DividerProps) => {
    return (
        <div className={styles.divider}>
            <h3>{title}</h3>
            <div className={styles.section} />
        </div>
    );
}

export default Divider;
