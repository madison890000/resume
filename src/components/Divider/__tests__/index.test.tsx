import React from 'react';
import Divider from '../index';
import renderer from 'react-test-renderer';

test('renders Divider', () => {
    const DividerDom = renderer.create(<Divider title="test Divider" />)
        .toJSON();
    expect(DividerDom)
        .toMatchSnapshot();
});
