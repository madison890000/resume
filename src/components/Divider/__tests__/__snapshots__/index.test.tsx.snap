// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Divider 1`] = `
<div
  className="divider"
>
  <h3>
    test Divider
  </h3>
  <div
    className="section"
  />
</div>
`;
